oauth2/src/main/java/com/okta/spring/oauth/OktaPropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -81,31 +81,35 @@
  *
  * @since 0.2.0
  */
-public class OktaPropertiesMappingEnvironmentPostProcessor implements EnvironmentPostProcessor {
+public class OktaPropertiesMappingEnvironmentPostProcessor implements EnvironmentPostProcessor, Ordered {
 
     private static final String OAUTH_CLIENT_PREFIX = "security.oauth2.client.";
     private static final String OAUTH_RESOURCE_PREFIX = "security.oauth2.resource.";
     private static final String OKTA_OAUTH_PREFIX = "okta.oauth2.";
 
-    private final Logger logger = LoggerFactory.getLogger(OktaPropertiesMappingEnvironmentPostProcessor.class);
-
     @Override
     public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
         environment.getPropertySources().addLast(remappedOktaToStandardOAuthPropertySource(environment));
-        environment.getPropertySources().addLast(discoveryPropertiesSource(environment));
-        environment.getPropertySources().addLast(loadYaml(new ClassPathResource("com/okta/spring/okta.yml")));
+        environment.getPropertySources().addLast(loadYaml(new FileSystemResource(new File(System.getProperty("user.home"), "okta/okta.yml")), false));
+        environment.getPropertySources().addLast(new DiscoveryPropertySource(environment));
+        environment.getPropertySources().addLast(oauthToClientPropertiesSource(environment));
+        environment.getPropertySources().addLast(loadYaml(new ClassPathResource("com/okta/spring/okta.yml"), true));
```

Comment:
Will this override the one from `user.home`? Also, should the `user.home` path be `.okta/okta.yml`?

Reply:
It should be `.okta/okta.yml`.  First one wins, so after the standard spring property sources (sys props, env vars, etc) these will get applied (the last one, the ClassPathResource contains any defaults)

Reply:
👍